core-web/libs/sdk/client/src/lib/utils/page/common-utils.ts

```diff
@@ -1,27 +1,36 @@
 interface PageRequestParamsProps {
     path: string;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    params: { [key: string]: any } | undefined; // QueryParams are typed as `any` in frameworks
+    params: { [key: string]: any } | undefined | URLSearchParams; // QueryParams are typed as `any` in frameworks
```

Comment:
Why use `any` and `undefined`? In case params will be optional, we could use `params?`?

Reply:
Not sure, we have to revisit all the typing to expose it in an external library. So we can leave it as it is for now